### PR TITLE
Added color and font for FieldHeadingObjects

### PR DESCRIPTION
**Pull request by michaelschwarz**

### CODE REVIEW

File: RptToXml/Enums.cs [hunk 1]
```diff
@@ -25,4 +25,4 @@ public enum ObjectTypes
 		ReportObject = 2 ^ 2,
 		All = Area & Section & ReportObject
 	}
-}
```
Comment (aidanjryan) [inline RptToXml/Enums.cs]: Was there supposed to be a change here?

File: RptToXml/RptDefinitionWriter.cs [hunk 2]
```diff
@@ -864,6 +864,15 @@ private void GetReportObjects(Section section, ReportDocument report, XmlWriter
 					writer.WriteAttributeString("FieldObjectName", fh.FieldObjectName);
 					writer.WriteAttributeString("MaxNumberOfLines", rasrdm_fh.MaxNumberOfLines.ToString());
 					writer.WriteElementString("Text", fh.Text);
+
+					if ((ShowFormatTypes & FormatTypes.Color) == FormatTypes.Color)
```
Comment (aidanjryan) [inline RptToXml/RptDefinitionWriter.cs]: Looks good, nice addition!

File: RptToXml/Enums.cs (hunk 1 above)
Comment (michaelschwarz) [reply RptToXml/Enums.cs]: No, my mistake

**Comment by ajryan** [comment]
Thanks for this!

**Pull request merged**